### ISSUE TITLE
Adding multiple custom chains

### ISSUE THREAD

**Issue by brucey0x** [peetzweg/vscode-crypto-address-lens#11]
Hey @peetzweg, thanks for creating this awesome extension!

I was wondering how I can expand the number of custom networks beyond the single custom and local RPC field that appear in settings? It'd be awesome if I could add the Ethereum and Arbitrum Goerli networks for example, and also have the UI/UX display the corresponding Etherscan links on mouse-over. Maybe inserting the relevant Etherscan domain could be a feature in the settings? 

Thank you.

**Comment by peetzweg** [comment]
Thanks @brucey0x, appreciate the feedback. Glad you find this extension also useful.

Fully agree that this would be nice. By now there are so many new relevant evm compatible chains that there is this need for adding more chains.

Happy to add this. Any idea how you would like to set this up in vscode? Are you using this custom rpc settings or just the hardcoded ones?

```
    "cryptoAddressLens.custom.rpc": "http://localhost:8545",
    "cryptoAddressLens.custom.enabled": false
```

I could either add more chains in this hardcoded fashion but it's not very customizable and future proof.
Opening it up a bit would be more future proof but error prone to configure for each user. 

Will revisit it probably this week.

https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema

**Comment by peetzweg** [comment]
@brucey0x just released v1.1.0 which offers more flexibility regarding configuration of rpcs and block explorers. You can now specify an arbitrary amount of rpcs and block explorers with these two configuration properties `cryptoAddressLens.rpcs` and `cryptoAddressLens.explorers`. 

See this section of the README.md to see the current defaults and how to add more:
https://github.com/peetzweg/vscode-crypto-address-lens#configuration
